objectherkenning_openbare_ruimte/on_edge/detection_pipeline/components/input_image.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from typing import Dict, List
+
+import cv2
+import numpy as np
+
+logger = logging.getLogger("image")
+
+
+class InputImage:
+    def __init__(self, image_full_path: str):
+        self.image = cv2.imread(str(image_full_path))
+        self.mapx = None
+        self.mapy = None
+
+    def resize(self, output_image_size: List):
```

Comment:
A `Tuple[int, int]` is more clear for image_size perhaps. That's what I use in other places as well.